spec/test-utils/test-data/generate-test-data.py

```diff
@@ -72,6 +72,30 @@ def main() -> None:
         master_private_key.public_key().public_bytes(Encoding.Raw, PublicFormat.Raw)
     )
 
+    b64_master_private_key = encode_base64(
+        master_private_key.private_bytes_raw()
+    )
+
+    self_signing_private_key = ed25519.Ed25519PrivateKey.from_private_bytes(
+        SELF_CROSS_SIGNING_PRIVATE_KEY_BYTES
+    )
+
+    b64_self_signing_public_key = encode_base64(
+        self_signing_private_key.public_key().public_bytes(Encoding.Raw, PublicFormat.Raw)
+    )
+
+    b64_self_signing_private_key = encode_base64(
+        self_signing_private_key.private_bytes_raw()
+    )
```

Comment:
suggest grouping the lines for each key together by removing the blank lines:

```suggestion
    self_signing_private_key = ed25519.Ed25519PrivateKey.from_private_bytes(
        SELF_CROSS_SIGNING_PRIVATE_KEY_BYTES
    )
    b64_self_signing_public_key = encode_base64(
        self_signing_private_key.public_key().public_bytes(Encoding.Raw, PublicFormat.Raw)
    )
    b64_self_signing_private_key = encode_base64(
        self_signing_private_key.private_bytes_raw()
    )
```